lib/xenops_server.ml

```diff
@@ -954,7 +954,7 @@ let rec atomics_of_operation = function
 			VM_create (id, None);
 		] @ [
 			VM_hook_script(id, Xenops_hooks.VM_pre_resume, Xenops_hooks.reason__none);
-			VM_restore (id, data);
+			VM_restore (id, data, false);
```

Comment:
Why not send a query on VM resume? I think we do need this. In fact, I don't believe that we need any of the changes in this file, and don't need to additional argument to the restore function.